### PR TITLE
Feat: Created Cards + minor changes in previous files

### PR DESCRIPTION
**Pull request by HeyNitin**
image
image

### CODE REVIEW

File: Components/Buttons/buttons.css [hunk 1]
```diff
@@ -28,4 +28,5 @@ a:any-link {
     height: 2rem;
     text-align: center;
     border-radius: 2rem;
+    z-index: 99;
```
Comment (rohanmathur91) [inline Components/Buttons/buttons.css]: Why so high z-index value?

File: Components/Cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+.wrapper {
+    display: flex;
+    gap: 2rem;
+    flex-wrap: wrap;
+}
+
+.card {
+
+    display: grid;
```
Comment (rohanmathur91) [inline Components/Cards/cards.css]: remove blank lines, use any tool to prettify the code.

File: Components/Avtars/avtars.html [hunk 3]
```diff
@@ -12,15 +12,15 @@
 <body>
     <div class="container">
 
-        <img class="avtar sm" src="../../Assets/imgs/male profile.jpg" alt="">
-        <img class="avtar md" src="../../Assets/imgs/male profile.jpg" alt="">
-        <img class="avtar lg" src="../../Assets/imgs/male profile.jpg" alt="">
+        <img class="avtar sm" src="../../Assets/imgs/male profile.jpg" alt="img">
+        <img class="avtar md" src="../../Assets/imgs/male profile.jpg" alt="img">
+        <img class="avtar lg" src="../../Assets/imgs/male profile.jpg" alt="img">
```
Comment (rohanmathur91) [inline Components/Avtars/avtars.html]:
``` suggestion
you can give a more meaningful alt eg avatar or profile
```

**Review by rohanmathur91 — COMMENTED** [review]
Add few comments, rest looks good to me

File: Components/Cards/cards.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          <i class="material-icons">favorite_border</i>
+          <i class="material-icons">share</i>
+          <i class="material-icons">list</i>
+        </footer>
+      </div>
+
+      <div class="card basicHeader shadow">
+        <img
+          src="https://static01.nyt.com/images/2021/11/30/sports/30messi-new/merlin_198571824_2182facc-eb3d-4ad6-a555-3cd35e63819a-superJumbo.jpg"
+          alt="img"
```
Comment (Maianki) [inline Components/Cards/cards.html]: try giving meaningful value to alt attribute.

**Review by Maianki — APPROVED** [review]
Suggested a minor change. Everything else looks good.

**Pull request merged**